client/src/containers/App/App.js

```diff
@@ -31,6 +33,9 @@ const App = () => (
         <li>
           <Link to="/users">Users</Link>
         </li>
+        <li>
+          <Link to="/VolunteersForm">Volunteers form</Link>
```

Comment:
I would change this to `<Link to="/volunteers">Volunteer</Link>`

... to be more consistent with the other links in the header

Reply:
yes that would be better
thanks